std_msgs/Header header
uint8 zone_height
uint8 zone_width
float32 max_dist
float32 min_dist
float32[] zone_distance_mm
float32[] zone_variance_mm
float32[] zone_reflectance